ext/mini_racer_extension/mini_racer_extension.cc

```diff
@@ -57,6 +57,8 @@ typedef struct {
     ContextInfo* context_info;
     Local<String>* eval;
     useconds_t timeout;
+    size_t max_mem;
+    size_t current_mem;
```

Comment:
Not your fault, but this struct is laid out poorly and will almost certainly waste a few bytes on 64-bit machines due to padding. Any 32-bit objects (such as timeout) should come after any 64-bit objects (such as the EvalResult* and these 2 size_t's). Again, not your fault, it was set up that way, just pointing it out ;p